### PR TITLE
[ADD] This apps lets you automatically generate attributes for products on a given category, this lets the user easily create its products catalog.

### PR DESCRIPTION
**Pull request by oscarolar**

### CODE REVIEW

File: product_category_attributes/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+#
+##############################################################################
+
+{
+    'name': 'Automatic Product Attributes',
+    'version': '1.0',
+    'category': 'Tools',
+    'description': """
+    Automated Action to Create Product Attributes
+=============================================
```
Comment (nhomar) [inline product_category_attributes/__openerp__.py]: This description will fail on rst.

Can you describe a little better, 2 or 3 screenshots can help.

You can use a README.md/rst files for better usage of this cool improvement.

File: product_category_attributes/data/server_actions.xml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+datas = []
+if categ_ids:
+    for categ in categ_obj.browse(cr, uid, categ_ids, context):
+        for attribute in categ.attribute_ids:
+            datas.append((0, 0, {'attribute_id':attribute.id}))
+        object.write({'attribute_line_ids': datas})]]>
+            </field>
+        </record>
+    </data>
+</openerp>
```
Comment (nhomar) [inline product_category_attributes/data/server_actions.xml:22]: It is a REALLY cool approach the usage of server actions.

Can you add a little test which trigger this action please, with that I think is complete all the approach and it is not quite difficult due to the on_create.

The test simply,

Create a Category with attribute.
Create a product on that actegory.
Assert the product create has the attribute.

Regards.

File: product_category_attributes/models/__init__.py [hunk 3]
```diff
@@ -0,0 +1 @@
+from . import product
```
Comment (nhomar) [inline product_category_attributes/models/__init__.py]: Add an extra blank line is necesary to avoid this red symbol here.

Comment (oscarolar) [inline product_category_attributes/models/__init__.py]: Done

**Comment by moylop260** [comment]
@oscarolar 
Can you send me the task number please to extract runbot link?

**Comment by oscarolar** [comment]
http://runbot.vauxoo.com/runbot/team/yoytec-12#8_0-yoytec-pr118 here is the runbot link, its already on green.

**Comment by moylop260** [comment]
@oscarolar 
Open ir.module view and search this module: 
local modules odoo

**Comment by moylop260** [comment]
@oscarolar,
My feedback:
products odoo

Branch name don't have SDGT info to know like test it but I tested it [here](http://youtu.be/IlMHYGiZAFA)

**Comment by oscarolar** [comment]
How to use according to the product owner  [VIDEO](https://www.youtube.com/watch?v=HAQPdnrFJiI&feature=youtu.be)
[SGDT](https://www.vauxoo.com/#id=3058&view_type=form&model=project.task&menu_id=602&action=778)

**Comment by oscarolar** [comment]
ENGLISH PLEASE, THEN WRITE

**Comment by moylop260** [comment]
@oscarolar 
Please, add i18n translation

**Comment by oscarolar** [comment]
Done

**Pull request merged**